src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallRecords/SdkFeatureBand.cs

```diff
@@ -4,7 +4,8 @@
 using System;
 using Microsoft.Deployment.DotNet.Releases;
 
-namespace Microsoft.DotNet.Workloads.Workload.Install
+#nullable disable
+namespace Microsoft.DotNet.Workloads.Workload.Install.InstallRecord
```

Comment:
Sorry I missed that in the beginning. The following method looks wrong. Release version is just a Version right? isn't comparison should be only the major, minor and build % 100.

There is also no validation. An SDK version band cannot be 5.0.1 or 5.0 only 5.0.1xx
public bool Equals(SdkFeatureBand other)
        {
            return ToString() == other.ToString();
        }
        public int CompareTo(SdkFeatureBand other)
        {
            return string.Compare(ToString(), other.ToString(), StringComparison.Ordinal);
        }